packages/scalecube-microservice/src/api/Router.ts

```diff
@@ -1,5 +1,12 @@
 import { Endpoint, RouteOptions } from '.';
 
+/**
+ * Allows to specify the logic of which instance of the required endpoint should be returned for the future call
```

Comment:
```
/**
 * Router - a way to Specify logic for the pick a remoteService
 *
 * `route` - logic to pick remoteService
 */
```

Reply:
The method that returns the appropriate remoteService for the provided search criteria. Returns null if no
appropriate remoteService was found